src/core/main.cpp

```diff
@@ -120,36 +120,36 @@ void printHelp()
 {
 	printf( "LMMS %s\n"
 		"Copyright (c) 2004-%s LMMS developers.\n\n"
-		"usage: lmms [ -r <project file> ] [ options ]\n"
+		"Usage: lmms [ -r <project file> ] [ options ]\n"
```

Comment:
`usage:` tends to be a lowercase word in other unix programs, as does the first letter of the sentence from what I'm observing.